ios/ZulipMobile/Info.plist

```diff
@@ -9,11 +9,11 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<string>Zulip</string>
```

Comment:
This should be updated in master but `PRODUCT_BUNDLE_IDENTIFIER` is usually a URL like `org.zulip.ZulipMobile`